src/main/java/com/diffblue/deeptestutils/Reflector.java

```diff
@@ -377,7 +376,7 @@ private static void makeFullyPublic(final CtClass cl) {
    *   and wrap it within an `InvocationTargetException`.
    *   TODO This currently only works for concrete classes, see TG-5895.
    */
-  public static <T> Object getInstance(final String className)
```

Comment:
I hadn't thought of this. Removed this commit so the public interface won't change.
Actually it would be nice if we could actually make this return something of type `T` and use generics instead of casts in generated tests... Something for a future PR.